README.md

```diff
@@ -508,21 +536,28 @@ start_termux_server() {
 
 audio_server() {
     select_distro || return 1
+    unmount_chroot
     echo -e "\nRunning Audio Server for $selected_distro..."
     start_termux_server
-    su -c "chroot-distro command $selected_distro \"export DISPLAY=:0 PULSE_SERVER=tcp:127.0.0.1:4713 && dbus-launch --exit-with-session && clear && mpd\""
+    su -c "chroot-distro command $selected_distro \"export DISPLAY=:0 PULSE_SERVER=tcp:127.0.0.1:4713 && dbus-launch --exit-with-session && mpd\""
+    clear
 }
 
 login_cli() {
     select_distro || return 1
+    unmount_chroot
```

Comment:
Why are you unmounting here? There should be no need for implicit unmounting as the `chroot-distro` script will (and should) not mount multiple times. The only case for unmounting would the clear the mounts after usage (there may be multiple calls to login). Also, there may be programs running from previous sessions, thus unmounting the mounts will only serve to wreck havoc within chroot.